components/user/UserSocials.js

```diff
@@ -0,0 +1,13 @@
+import Icon from "../Icon";
+
+function UserSocial({ social }) {
+    console.log({ social })
+    return (
+        <a href={social.url} className="w-8 h-8 hover:scale-125 transition ease-in-out delay-100">
+            <Icon name={social.platform} />
+        </a>
+
+    )
+}
+
+export default UserSocial
```

Comment:
- no blank line ending indicated by red mark on github
- not using project standards, missing `;`